Workade/Views&ViewModels/Home/CollectionView/OfficeCollectionViewCell.swift

```diff
@@ -8,39 +8,27 @@
 import UIKit
 
 class OfficeCollectionViewCell: UICollectionViewCell {
-    private lazy var backgroundImageView: UIImageView = {
-        let imageView = UIImageView()
-        imageView.contentMode = .scaleAspectFill
-        imageView.clipsToBounds = true
-        imageView.layer.cornerRadius = 12
-        imageView.backgroundColor = .theme.groupedBackground // Skeleton color
+    private lazy var backgroundImageView: CellImageView = {
+        let imageView = CellImageView(bounds: bounds)
         imageView.translatesAutoresizingMaskIntoConstraints = false
         
-        let path = UIBezierPath(rect: self.bounds)
-        let blackLayer = CAShapeLayer()
-        blackLayer.path = path.cgPath
-        blackLayer.fillColor = UIColor.black.withAlphaComponent(0.1).cgColor
-        imageView.layer.addSublayer(blackLayer)
-        
         return imageView
     }()
     
     private let mapButton: UIButton = {
         let button = UIButton()
-        let config = UIImage.SymbolConfiguration(pointSize: 16, weight: .bold)
-        let image = UIImage(systemName: "map", withConfiguration: config)
+        let image = SFSymbol.mapInCell.image
         button.setImage(image, for: .normal)
-        button.tintColor = .white
         button.layer.cornerRadius = 18
-        button.layer.backgroundColor = UIColor.black.withAlphaComponent(0.2).cgColor
+        button.layer.backgroundColor = UIColor.black.withAlphaComponent(0.4).cgColor
         button.translatesAutoresizingMaskIntoConstraints = false
         
         return button
     }()
     
     private let regionNameLabel: UILabel = {
         let label = UILabel()
-        label.font = .customFont(for: .subHeadline)
+        label.font = .systemFont(ofSize: 15, weight: .bold)
```

Comment:
```suggestion
        label.font = .customFont(for: .subHeadline)
```

[P2] 위와 동일하게 수정해주시면 감사하겠습니다!!